msal/src/Microsoft.Identity.Client/Telemetry.cs

```diff
@@ -65,7 +65,7 @@ public void RegisterReceiver(Receiver r)
         /// <returns></returns>
         public static Telemetry GetInstance()
```

Comment:
>GetInstance() [](start = 32, length = 13)

implemented as a method... then it's called GetInstance. This is the most used pattern in the codebase. pls. stick to this one

Reply:
#Resolved